<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1361</width>
    <height>613</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="Window">
   <property name="styleSheet">
    <string notr="true">
QLabel{color:white;}
QTableWidget{background:white;}
QTableWidget::item {
    background-color: white;
	color:black;
}</string>
   </property>
   <widget class="QLabel" name="kamera_ekrani">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>120</y>
      <width>810</width>
      <height>470</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>470</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>810</width>
      <height>470</height>
     </size>
    </property>
    <property name="font">
     <font>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #354152;</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="vardiya_listesi">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>120</y>
      <width>571</width>
      <height>471</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Öğrencilerin Bulunduğu Liste&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::SelectedClicked</set>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>işçi  No</string>
     </property>
     <property name="font">
      <font>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Adı Soyadı</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Giriş Tarihi</string>
     </property>
     <property name="font">
      <font>
       <stylestrategy>PreferDefault</stylestrategy>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Çıkış Tarihi</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tarih</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="isi_info">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>120</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>80</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #354152;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <widget class="QLabel" name="labelGirisCikis_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>5</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0%)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Adı Soyadı :&lt;/p&gt;&lt;p&gt;İşçi No :&lt;/p&gt;&lt;p&gt;Vardiyası :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_adiSoyadi">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>8</y>
       <width>211</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Adı Soyadı</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_isciNo">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>45</y>
       <width>181</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>İşçi No</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_girilenVardya">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>191</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Vardiyası</string>
     </property>
    </widget>
    <widget class="QLabel" name="profile_picture">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>personReal.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="ders_info">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>551</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: #354152;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <widget class="QLabel" name="labelGirisCikis_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>3</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0,0,0,0%)</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Vardiya Adı :&lt;/p&gt;&lt;p&gt;Vardiya Kodu :&lt;/p&gt;&lt;p&gt;Vardiya Mevcudu :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_vardiyaKodu">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>45</y>
       <width>231</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Vardiya Kodu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_vardiyaMevcudu">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>231</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Vardiya Mevcudu</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_vardiyaAdi">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>5</y>
       <width>231</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Vardiya Adı</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1361</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionStartCam"/>
    <addaction name="actionStopCam"/>
    <addaction name="actionGirisiBitir"/>
   </widget>
   <widget class="QMenu" name="menuEkle">
    <property name="title">
     <string>Ekle</string>
    </property>
    <addaction name="IsciEkle"/>
    <addaction name="VardiyasorumlusuEkle"/>
    <addaction name="DersEkle"/>
   </widget>
   <widget class="QMenu" name="menuAyarlar">
    <property name="title">
     <string>Ayarlar</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_IsciListesi"/>
    <addaction name="action_OgrenciAra"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuEkle"/>
   <addaction name="menuAyarlar"/>
   <addaction name="menuExit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStartCam">
   <property name="text">
    <string>Start Camera</string>
   </property>
  </action>
  <action name="actionStopCam">
   <property name="text">
    <string>Stop Camera</string>
   </property>
  </action>
  <action name="IsciEkle">
   <property name="text">
    <string>Öğrenci Ekle</string>
   </property>
  </action>
  <action name="VardiyasorumlusuEkle">
   <property name="text">
    <string>Öğretmen Ekle</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Modelleri Yükle</string>
   </property>
  </action>
  <action name="action6">
   <property name="text">
    <string>deneme6</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="DersEkle">
   <property name="text">
    <string>Ders Ekle</string>
   </property>
  </action>
  <action name="action_IsciListesi">
   <property name="text">
    <string>Öğrenci Listesi</string>
   </property>
  </action>
  <action name="action_OgrenciAra">
   <property name="text">
    <string>Öğrenci Ara</string>
   </property>
  </action>
  <action name="actionGirisiBitir">
   <property name="text">
    <string>Yoklamayı Bitir</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
